x/launch/client/cli/tx_create_chain.go

```diff
@@ -58,6 +62,23 @@ func CmdCreateChain() *cobra.Command {
 				if err != nil {
 					return err
 				}
+				initialGenesis = types.NewGenesisURL(genesisURL, genesisHash)
+			}
+
+			// parse genesis config file for initialGenesis
+			genesisConfigFile, err := cmd.Flags().GetString(flagGenesisConfigFile)
+			if err != nil {
+				return err
+			}
+			if genesisConfigFile != "" {
+				if args[1] == "" {
+					return errors.New("genesis config file requires a source url to be set")
+				}
+				initialGenesis = types.NewConfigGenesis(genesisConfigFile)
+			}
+
+			if genesisURL != "" && genesisConfigFile != "" {
```

Comment:
I don't think we need to return an error. If no option is specified then `DefaultInitialGenesis` is used

Reply:
This is catching the case where a user is setting BOTH genesisURL and Config file.  This is just enforcing that they can only set one or the other - not both, which is invalid.  The default case is still `DefaultInitialGenesis`.

Reply:
added a comment to clarify